storage/storage.hpp

```diff
@@ -83,9 +83,14 @@ class Storage
   // country were successfully downloaded.
   TUpdate m_update;
 
+  // If |m_dataDir| is not empty Storage will create version directories and download maps into
```

Comment:
Are you sure about it? I guess the both variant is valid. 
I fixed to save time.